tests/test_main.py

```
@@ -81,6 +82,99 @@ def _mock_run_with_keyboard_interrupt(coro: Any) -> None:
+    def submit(self, func, *args, **kwargs):
+        if getattr(func, "__name__", "") == "_close_meshtastic":
+            if self.submit_timeout:
+                raise concurrent.futures.TimeoutError()
+            if self.close_future_factory is not None:
+                self.future = self.close_future_factory()
+                return self.future
+
+        future = concurrent.futures.Future()
+        try:
+            result = func(*args, **kwargs)
+        except Exception as exc:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)